plugins/inputs/knx_listener/knx_listener.go

```diff
@@ -0,0 +1,149 @@
+package knx_listener
+
+import (
+	"log"
+
+	"github.com/vapourismo/knx-go/knx/dpt"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type address_target struct {
```

Comment:
Go typically avoids underscores in type names, though this is less important for private types.

Reply:
Changed to camel-case.